content/docs/setup/kubernetes/platform-setup/gke/index.md

```diff
@@ -17,6 +17,11 @@ Follow these instructions to prepare an GKE cluster for Istio.
       --project <project-id>
     {{< /text >}}
 
+    - **NOTE** to use the Istio CNI feature, the
```

Comment:
Remove **NOTE** and instead use:

> {{< warning_icon >}}

This will indent the paragraph and add a warning icon.

Reply:
done